src/pages/developers/tutorials.js

```diff
@@ -241,11 +240,7 @@ const TutorialsPage = ({ data, pageContext }) => {
 
   const allTutorials = []
     .concat(externalTutorials, internalTutorials)
-    .filter((tutorial) =>
-      hasTutorials(pageContext.language)
-        ? tutorial.lang === pageContext.language
-        : tutorial.lang === "en"
-    )
+    .filter((tutorial) => tutorial.lang === pageContext.language)
```

Comment:
I don't think this logic quite gives us what we want. If a language doesn't have any tutorials, we'll now display 0 tutorials for that language, e.g.
![Image 2021-11-23 at 9 56 08 PM](https://user-images.githubusercontent.com/8097623/143172793-1ddf0e28-fcdf-4c5b-8890-cc11cc1e48c9.jpg)

What we want is to:
1) if the language has translated tutorials, display those
2) if the language doesn't have translated tutorials, display our English tutorials

I feel like we discussed just refactoring the `hasTutorials` to be a helper function in this file that iterates through the tutorials & to return `true` as soon as it hits one in that native language. If it doesn't find one, return `false`. Basically lodash's `any` or `some`: https://lodash.com/docs/4.17.15#some

Reply:
(we have lodash installed in the project, so you should be able to use it).

Reply:
Ah gotcha, my mistake. Will get that added in now!